README.md

```diff
@@ -167,20 +167,16 @@ Note: This tutorial requires that the Azure SQL database is setup as shown in [C
 - In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code.
 
     ```csharp
-    public static class HttpTriggerCSharp1
+    public static async Task<IActionResult> Run(
```

Comment:
Why did you remove the `[FunctionName]` attribute?

Reply:
The wording in the readme says to replace the Run block in the code. The [FunctionName] exists outside of this block

Reply:
Made a change yesterday to the doc to have a happy landing url for the input binding. When updating the code in the readme block, I copied over a bit too much of the code. This PR is to get it to realign with the wording, "In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code."